test/unit/mem/copy/src/BufSlicing.cpp

```diff
@@ -46,42 +45,37 @@ struct TestContainer
     {
     }
 
-
     auto createHostBuffer(Vec extents, bool indexed) -> BufHost
     {
         BufHost bufHost(alpaka::allocBuf<TData, TIdx>(devHost, extents));
         if(indexed)
         {
-            TData* const ptr = alpaka::getPtrNative(bufHost);
-            for(TIdx i(0); i < extents.prod(); ++i)
+            TData* const ptr = bufHost.data();
```

Comment:
Why do you need to replace `getPtrNative` by `.data()`?

Reply:
I don't _need_ to, but it's shorter and it follows the approach used in STL containers.